lib/ansible/plugins/lookup/template.py

```diff
@@ -36,6 +36,7 @@ class LookupModule(LookupBase):
     def run(self, terms, variables, **kwargs):
 
         convert_data_p = kwargs.get('convert_data', True)
+        extra_vars = kwargs.get('vars', {})
```

Comment:
from core meeting, the vote is in and we would like the new option to be named `template_vars`